cartography/data/permission_relationships.yaml

```diff
@@ -1,18 +1,29 @@
-# map principals who can read items in a S3Bucket
+# Map principals that can read items from an S3Bucket. Specifically,
+# create an (:AWSPrincipal)-[:CAN_READ]->(:S3Bucket) relationship
+# for principals that have a policy attached with _any_ of the below
+# permissions. Similar logic applies for the other entries in this file.
 - target_label: S3Bucket
   permissions:
   - S3:GetObject
   relationship_name: CAN_READ
-# map principals who can write items in a S3Bucket
+# Map principals that can write items to an S3Bucket.
 - target_label: S3Bucket
   permissions:
   - S3:PutObject
   relationship_name: CAN_WRITE
-# map principals retrieve data fron dynamodb
+# Map principals that can retrieve data from DynamoDB.
 - target_label: DynamoDBTable
   permissions:
   - dynamodb:BatchGetItem
   - dynamodb:GetItem
   - dynamodb:GetRecords
   - dynamodb:Query
   relationship_name: CAN_QUERY
+# Map principals that can administer RedshiftClusters.
```

Comment:
I realize that not every user may have a use for Redshift, but I think the perf impact to having this in here is almost negligible and it's also helpful to show a wildcard action.